docs/user/faq.rst

```diff
@@ -889,3 +889,35 @@ the tutorial in the docs provides an excellent introduction to
 `testing Falcon apps with pytest <http://falcon.readthedocs.io/en/stable/user/tutorial.html#testing-your-application>`_.
 
 (See also: `Testing <http://falcon.readthedocs.io/en/stable/api/testing.html>`_)
+
+How to set cookies in simulate request for testing?
+---------------------------------------------------
+
+This can be done by setting ``headers={'Cookies': 'xxx=yyy'}`` in ``simulate_request``. Here is an example:
```

Comment:
`Cookies` ==> `Cookie`

Reply:
Also, I think it would be helpful to explain how to include multiple cookie values per https://tools.ietf.org/html/rfc6265#section-5.4 .

Reply:
Solved.